packages/cli/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "opticks-cli",
-  "version": "1.0.0",
+  "version": "0.0.1",
```

Comment:
Corrects this as the [version on npm](https://www.npmjs.com/package/opticks-cli) is 0.0.1